pkg/kubelet/cm/devicemanager/plugin/v1beta1/stub.go

```diff
@@ -144,13 +149,33 @@ func (m *Stub) Start() error {
 	return nil
 }
 
+func (m *Stub) Restart() error {
+	klog.InfoS("Restarting Device Plugin server")
+	if m.server == nil {
+		return nil
+	}
+	// Send terminate signal to ListAndWatch()
+	m.termSignal <- true
+
+	m.server.Stop()
+	m.server = nil
+
+	m.Start()
+	return m.Start()
```

Comment:
(I probably commented already, sorry for the repetition) why call `m.Start()` twice?

Reply:
Might have been a copy paste error, will fix in the next upload. Thanks for pointing this out.